HISTORY.md

```diff
@@ -18,6 +18,7 @@
 ## 6.1.0 (3/27/2019)
 ### New Features
 * Introduce two more stats levels, kExceptHistogramOrTimers and kExceptTimers.
+* Added BottommostLevelCompaction::kForceOptimized to avoid double compacting newly compacted files in bottom level compaction of manual compaction.
```

Comment:
For my understanding, is it an option because `kForce` should be able to force recompaction even on files that were trivially moved to the bottom level?

Reply:
Yes, we decided to make it an option because tools like `option_change_migration` and tests like `DBCompactionTestWithParam .ConvertCompactionStyle` expects manual compactions to generate one single file in the bottom level